examples/mlmc.cpp

```diff
@@ -273,19 +273,25 @@ int main(int argc, char* argv[])
 
     // Create Upscaler and Solve
     unique_ptr<FiniteVolumeMLMC> fvupscale;
+    UpscaleParameters param;
```

Comment:
How do you think about initializing `UpscaleParameters` before command line option parser, and then put the relevant member of `param` (e.g. `param.spect_tol`) directly in `OptionParser`? Something like
```
args.AddOption(&(param.spect_tol), "t", ...
```
This way we can save a few lines initializing each of the parameters in the beginning of `main`, as well as the lines below that assign values. And this comment is relevant for most of the examples/tests files.

Reply:
I like this idea, and I've taken it one step further. Let me know what you think.

Reply:
The register function looks good to me (it saves another few lines in each example/test). My only concern would be that new users may be not aware of these command options when looking at the cpp files. Some documentation may help?